src/RecoveryServices/RecoveryServices/Vault/NewAzureRMRecoveryServicesVault.cs

```diff
@@ -53,10 +53,25 @@ public class NewAzureRmRecoveryServicesVault : RecoveryServicesCmdletBase
         [ValidateNotNullOrEmpty]
         public string Location { get; set; }
 
+        /// <summary>
+        /// Gets or sets the tags of the vault
+        /// </summary>
         [Parameter(Mandatory = false)]
         [ValidateNotNullOrEmpty]
         public Hashtable Tag { get; set; }
 
+        /// <summary>
+        /// Enables or disables classic alerts for RS vault.
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        public bool? DisableClassicAlerts { get; set; }
```

Comment:
Please use singular parameter name 
```suggestion
        public bool? DisableClassicAlert { get; set; }
```